src/server/create-renderer.js

```diff
@@ -23,7 +23,7 @@ export type RenderOptions = {
   directives?: Object;
   isUnaryTag?: Function;
   cache?: RenderCache;
-  template?: string;
+  template?: string | (content: string, context: any) => string;
```

Comment:
Wouldn't `(context, content)` be a better API, consistent with, for example, vuex actions, which have `context` as their first argument?